### PR TITLE
[classify] Migrate to new client LanguageServiceClient in 0.16.2

### PR DESCRIPTION
**Pull request by gogasca**

### CODE REVIEW

File: language/language.php [hunk 1]
```diff
@@ -290,7 +290,7 @@
         // Regex to match a Cloud Storage path as the first argument
         // e.g "gs://my-bucket/file_with_text.txt"
         if (preg_match('/^gs:\/\/([a-z0-9\._\-]+)\/(\S+)$/', $content, $matches)) {
-            classify_text_from_file($content, $projectId);
+            classify_text_from_file($matches[0], $projectId);
```
Comment (bshaffer) [inline language/language.php]: In this case, `$content` will always be equal to `$matches[0]`. Why update this?

File: language/src/classify_text.php [hunk 2]
```diff
@@ -47,21 +47,23 @@ function classify_text($text, $projectId = null)
...
+        // Pass GCS URI and set document type to PLAIN_TEXT
+        $document->setContent($text)->setType(1);
+        // Call the analyzeSentiment function
+        $response = $languageServiceClient->classifyText($document);
+        $categories = $response->getCategories();
+        // Print document information
+        foreach ($categories as $category) {
+            printf('Category Name: %s' . PHP_EOL, $category->getName());
+            printf('Confidence: %s' . PHP_EOL, $category->getConfidence());
+            printf(PHP_EOL);
```
Comment (bshaffer) [inline language/src/classify_text.php]: nit: just use `print` here

File: language/src/classify_text_from_file.php [hunk 3]
```diff
@@ -36,22 +37,25 @@
...
-    foreach ($categories as $category) {
-        printf('Category Name: %s' . PHP_EOL, $category['name']);
-        printf('Confidence: %s' . PHP_EOL, $category['confidence']);
-        printf(PHP_EOL);
+    $languageServiceClient = new LanguageServiceClient(['projectId' => $projectId]);
+    try {
+        // Create a new Document
+        $document = new Document();
+        // Pass GCS URI and set document type to PLAIN_TEXT
+        $document->setGcsContentUri($gcsUri)->setType(1);
```
Comment (bshaffer) [inline language/src/classify_text_from_file.php]: let's use an actual call to the Enum class here

File: language/src/classify_text.php [hunk 4]
```diff
@@ -47,21 +47,23 @@ function classify_text($text, $projectId = null)
...
-    foreach ($categories as $category) {
-        printf('Category Name: %s' . PHP_EOL, $category['name']);
-        printf('Confidence: %s' . PHP_EOL, $category['confidence']);
-        printf(PHP_EOL);
+    $languageServiceClient = new LanguageServiceClient(['projectId' => $projectId]);
+    try {
+        // Create a new Document
+        $document = new Document();
+        // Pass GCS URI and set document type to PLAIN_TEXT
+        $document->setContent($text)->setType(1);
```
Comment (bshaffer) [inline language/src/classify_text.php]: let's use an actual call to the Enum class here

**Comment by gogasca** [comment]
Changes submitted, thanks for the review

**Pull request merged**